docker/compose/docker-compose.yml

```diff
@@ -81,6 +81,8 @@ services:
     command: segmentstore
     environment:
       WAIT_FOR: bookie1:3181,bookie2:3182,bookie3:3183,hdfs:8020
+      ENABLE_HDFS: "true"
```

Comment:
This is the same discussion we have having in other comment thread. Will update here according to the outcome of that discussion.